runtime/doc/lua.txt

```diff
@@ -913,6 +913,54 @@ vim.fn.{func}({...})					*vim.fn*
         Note: vim.fn keys are generated lazily, thus `pairs(vim.fn)` only
         enumerates functions that were called at least once.
 
+vim.define_augroup({group_name}, {clear_autocmds})       *vim.define_augroup()*
```

Comment:
can we use `vim.augroup_define` and `vim.augroup_undefine()` instead? This would align with `sign_define()` and `sign_undefine()`